imdb/SearchResults.py

```diff
@@ -0,0 +1,131 @@
+import logging
+import sys
+import traceback
+
+import requests
+
+from bs4 import BeautifulSoup
+
+
+def get_search_movies(movie):
+
+    url = "http://www.imdb.com/find?s=tt&q=" + movie
+    try:
+        r = requests.get(url)
+    except requests.exceptions.RequestException as e:
+        logging.info(e)
+        traceback.print_exc(e)
+        json_object = {}
+        json_object['error'] = "Service Unavailable"
+        tup = (True, 503, json_object)
```

Comment:
correct
there are more, have a look(http://www.restapitutorial.com/httpstatuscodes.html), no need to remember